lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -50,19 +50,19 @@
     default: "/"
   cert_chain:
     description:
-      - The CA certificate chain in PEM encoded format.
-      - Note that prior to 2.4, this parameter expected a path to a file.
-        Since 2.4 this is now accomplished using a lookup plugin. See examples for detail.
+      - The path to the CA certificate chain in PEM encoded format.
+        As of 2.4 this can also be accomplished using a lookup plugin.
```

Comment:
I don't think this needs to necessarily mention lookup anymore because we're not forcing people to use content and they can just pass the file if they have it.   I believe the comment was only added because the module went from supporting only "content" to supporting only paths.  

Perhaps we're better off saying that as of 2.4, both content and paths are supported;  And if the param is not a path it is assumed to be content.